subscription/subscription_worker.py

```diff
@@ -0,0 +1,137 @@
+import boto3
+import json
+import multiprocessing
+import os
+import time
+from flask import Flask, jsonify
+from sys import stdout
+from botocore.exceptions import ClientError
+
+AWS_REGION = os.getenv("AWS_REGION")
+QUEUE_URL = os.getenv("QUEUE_URL")
+DEAD_LETTER_QUEUE_URL = os.getenv("DEAD_LETTER_QUEUE_URL")
+SUB_DEAD_LETTER_QUEUE_URL = os.getenv("SUB_DEAD_LETTER_QUEUE_URL")
+LONG_POLL_TIME = os.getenv("LONG_POLL_TIME")
+SNS_NAME = os.getenv("SNS_NAME")
+
+class sns:
```

Comment:
python class naming convention is CapWords

Reply:
Also, I am leaning towards us separating classes into their own files for cleaner file org

Reply:
I have changed the class name to Sns and I have moved it to a new file called sns.py